test/mbr_update_test.rb

```diff
@@ -20,6 +20,9 @@
         dev + num.to_s
       end
 
+      # by default common architecture"
+      allow(Yast::Arch).to receive(:architecture).and_return("x86_64")
```

Comment:
yeah, it also helps with other modules that can be called here to have unified env.